test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -210,7 +207,7 @@ await CliTester.Run(new RunConfiguration[]
                         "start --port 7073 --verbose"
                     },
                     ExpectExit = false,
-                    Test = async (workingDir, p) =>
+                    Test = async (workingDir, p, _) =>
                     {
                         using (var client = new HttpClient() { BaseAddress = new Uri("http://localhost:7073") })
```

Comment:
should we also try to kill 7073 when disposing? Or whatever port the process is running on?

Reply:
Yeah, it looks like I forgot to utilize "HostProcessPort" for the tests not using the default - I'll go through and update them all